packages/core/test/Cardano/types/Cip1854ExtendedAccountPublicKey.test.ts

```diff
@@ -0,0 +1,61 @@
+import { Bip32PublicKeyHex } from '@cardano-sdk/crypto';
+import { Cip1854ExtendedAccountPublicKey } from '../../../src/Cardano';
+
+const bip32PublicKeyPrefix = 'acct_shared_xvk';
+
+describe('Cardano/types/Cip1854ExtendedAccountPublicKey', () => {
+  const publicKeyHex = Bip32PublicKeyHex(
+    '979693650bb44f26010e9f7b3b550b0602c748d1d00981747bac5c34cf5b945fe01a39317b9b701e58ee16b5ed16aa4444704b98cc997bdd6c5a9502a8b7d70d'
+  );
+
+  describe('Cip1854ExtendedAccountPublicKey', () => {
+    it('Accepts a valid bech32-encoded CIP1854 public key', () => {
+      const bip32PublicKey =
+        'acct_shared_xvk1q395kywke7mufrysg33nsm6ggjxswu4g8q8ag7ks9kdyaczchtemd5d2armrfstfa32lamhxfl3sskgcmxm4zdhtvut362796ez4ecqx6vnht';
+      expect(() => Cip1854ExtendedAccountPublicKey(bip32PublicKey)).not.toThrow();
+    });
+
+    it('Throws an error when an invalid bech32-encoded CIP1854 public key is passed', () => {
+      expect(() => Cip1854ExtendedAccountPublicKey(publicKeyHex)).toThrow();
+      expect(() =>
+        Cip1854ExtendedAccountPublicKey(
+          'addr_test1qpfhhfy2qgls50r9u4yh0l7z67xpg0a5rrhkmvzcuqrd0znuzcjqw982pcftgx53fu5527z2cj2tkx2h8ux2vxsg475q9gw0lz'
+        )
+      ).toThrow();
+      expect(() => Cip1854ExtendedAccountPublicKey('invalid')).toThrow();
+    });
+  });
+
+  describe('fromBip32PublicKeyHex', () => {
+    it('Encodes a valid CIP1854 bip32 public key according to CIP5 specification', () => {
+      const cip1854PublicKey = Cip1854ExtendedAccountPublicKey.fromBip32PublicKeyHex(publicKeyHex);
+      expect(cip1854PublicKey.startsWith(bip32PublicKeyPrefix)).toBe(true);
+    });
+
+    it('Throws an error when an invalid hex string is passed', async () => {
+      expect(() => Cip1854ExtendedAccountPublicKey.fromBip32PublicKeyHex(Bip32PublicKeyHex('invalid'))).toThrow();
+      expect(() =>
+        Cip1854ExtendedAccountPublicKey.fromBip32PublicKeyHex(
+          Bip32PublicKeyHex('701e58ee16b5ed16aa4444704b98cc997bdd6c5a9502a8b7d70d')
```

Comment:
`Bip32PublicKeyHex` is what throws here, not `Cip1854ExtendedAccountPublicKey.fromBip32PublicKeyHex`, so it's not testing the appropriate subject. I suggest removing hex length validation from `fromBip32PublicKeyHex` because `Bip32PublicKeyHex` is already validated

Reply:
Fixed on 7b583d8

Reply:
It's still validating the length [here](https://github.com/input-output-hk/cardano-js-sdk/pull/1346/files#diff-f097170118dd093d5923cad70352a6070b0adc8ad213e5c0c06bbbc02b8badceR22-R26)

Reply:
Also still testing `Bip32PublicKeyHex` [here](https://github.com/input-output-hk/cardano-js-sdk/pull/1346/files#diff-82112a97b58a6472f92ac1d7c91f6d618b21f04e545876d13c01e9e58ab918c7R36)

Reply:
Should I remove [this](https://github.com/input-output-hk/cardano-js-sdk/blob/7b583d8fa510972a8731f68fba1c7245e0be9b27/packages/core/src/Cardano/types/Cip1854ExtendedAccountPublicKey.ts#L22-L26) entire assertion block since it's already validated by `Bip32PublicKeyHex`?